lib/NetworkManager.js

```diff
@@ -178,15 +174,13 @@ class NetworkManager extends EventEmitter {
       }).catch(debugError);
     }
 
-    const requestHash = generateRequestHash(event.request);
-    const requestId = this._requestHashToRequestIds.firstValue(requestHash);
-    if (requestId) {
+    const requestId = event.requestId;
+    if (requestId && this._requestIdToRequestWillBeSentEvent.has(requestId)) {
```

Comment:
how can there be no `requestId`?

Reply:
Browser initiated requests which we don't have instrumentation for, like the opensearch thing, don't have a devtools request id in the mojo request struct and therefore don't get a requestId in the requestIntercepted event